app/controllers/api/v1/menu_items_controller.rb

```diff
@@ -5,9 +5,21 @@ def index
     render json: MenuItemSerializer.new(@restaurant.menu_items)
   end
 
+  def update
+    if !params[:name].present? || !params[:description].present? || !params[:category].present? || !params[:price].present? || !params[:tags].present?
+      render json: { error: "One or more attributes is missing" }
```

Comment:
I have a similar sad path response on my create menu item action, I think we should consider moving these to an error serializer if/when we refactor